example/webgpu.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>ThorVG Lottie Player</title>
+    <meta name="description" content="A web lottie player using ThorVG as a renderer" />
+    <script src="../dist/lottie-player.js"></script>
+  </head>
+  <body>
+    <div style="margin-top: 60px;">
+      <div>
+        <button onclick="playAnimation()">Play</button>
+        <button onclick="pauseAnimation()">Pause</button>
+        <button onclick="stopAnimation()">Stop</button>
+        <button onclick="createAnimation()">Create</button>
+        <button onclick="destroyAnimation()">Destroy</button>
+        <button onclick="terminateModule()">Terminate</button>
```

Comment:
How about pairing initializeModule() with terminateModule()?
Alternatively, hide them within createAnimation() and destroyAnimation().

Reply:
We don't need to call `init()` since it's called from import statement. Only needed is call a `term()` when player is no longer displaying from the screen.

This example shows single animation, combining term() with destroyAnimation would make sense!